### ISSUE TITLE
Automatically email CSV reports to all program admins at one week intervals.

### ISSUE THREAD

**Issue by Miguel-314** [civiform/civiform#5553]
Is it possible to automatically send CSV reports over email to all listed program admins? Emails could go out at pre set intervals for example once a day or once a week. This item is a "nice to have" raised by SHA program admin.

**Comment by bion** [comment]
Yes, the system supports recurring scheduled work. Would it be ok to link to the CSV download URL instead of attaching the CSV to the email? The former is more secure.

**Comment by Miguel-314** [comment]
Yes, I think that would still be an improvement to current business process.
